File: MAY99_3/MAY99_3/MAY99_3.cpp
#include<stdio.h>
int gcd(int a, int b){
	if(b==0)
		return a;
	return gcd(b, a%b);
}
int main(){
	int t;
	scanf("%i", &t);
	for(int i=0; i<t; i++){
		int x, y, z;
		scanf("%i %i %i", &x, &y, &z);
		if(z>x && z>y)
			printf("NO\n");
		else{
			int g=gcd(x,y);
			if(z%g==0)
				printf("YES\n");
			else
				printf("NO\n");
		}
	}
}

File: MAY99_3/MAY99_3/MAY99_3.py
import fractions
t=int(input())
while(t>0):
	the_string = raw_input()
	x, y, z= the_string.split()
	x=int(x)
	y=int(y)
	z=int(z)
	if(z > x and z > y):
		print("NO")
	else:
		g = fractions.gcd(x,y)
		if(z % g == 0):
			print("YES")
		else:
			print("NO")
	t=t-1